.github/workflows/test.yml

```diff
@@ -1,21 +1,19 @@
 name: Tests
 
-on:
-  push:
-  pull_request:
-  schedule:
-    - cron: '0 0 * * *'
+on: [push, pull_request]
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: true
       matrix:
+        os: [ubuntu-latest]
         php: [7.3, 7.4, 8.0, 8.1]
-        illuminate_version: [6.*, 8.*]
-        composer_flags:  ['', '--prefer-lowest']
+        illuminate_version: [6.*, 7.*, 8.*, 9.*]
+        stability: [prefer-lowest, prefer-stable]
```

Comment:
`--prefer-stable` is not the opposite of `--prefer-lowest`, it also makes sense in combination.

Actually, I think that `--prefer-stable` is even the default, but I'm not totally sure.

Reply:
Indeed, both can be used in combination I think. `prefer-stable` will check for packages with `minimum-stability` set to stable. But I don't think it's the default (and if so, it's still explicit this way). It's just an extra check to see if this package is easily installable, feel free to improve if possible!